components/table/table-col-sort-button.js

```diff
@@ -37,30 +37,47 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 				align-items: center;
 				background-color: transparent;
 				border: none;
+				box-sizing: border-box;
 				color: inherit;
 				cursor: pointer;
 				display: inline-flex;
 				font-family: inherit;
 				font-size: inherit;
+				height: var(--d2l-sortable-button-height);
 				letter-spacing: inherit;
+				line-height: 0.9rem;
 				margin: 0;
-				padding: 0;
+				padding: 0.75rem;
 				text-decoration: none;
+				width: var(--d2l-sortable-button-width);
 			}
 			button::-moz-focus-inner {
 				border: 0;
 			}
 			button:disabled {
 				opacity: 0.5;
 			}
-			button:hover {
-				text-decoration: underline;
-			}
 			button:focus-visible,
 			button:${unsafeCSS(getFocusPseudoClass())} {
-				border-radius: 0.2rem;
+				border-radius: var(--d2l-sortable-button-border-focus-radius);
 				box-shadow: 0 0 0 2px #ffffff, 0 0 0 4px var(--d2l-color-celestine);
+				height: calc(100% - 8px);
+				margin-inline-start: 4px; /* Used to offset the outer box shadow */
 				outline-style: none;
+				padding-inline-start: calc(0.75rem - 4px);
```

Comment:
If you use the `--d2l-table-cell-padding` var, you should also use it here for this calculation.